map_view/display-layer-view-state/src/main/java/com/esri/samples/display_layer_view_state/DisplayLayerViewStateSample.java

```diff
@@ -29,116 +36,142 @@
 
   private MapView mapView;
-
-  private static final int MIN_SCALE = 40000000;
-  private static final int TILED_LAYER = 0;
-  private static final int IMAGE_LAYER = 1;
-  private static final int FEATURE_LAYER = 2;
-
-  private static final String SERVICE_TIME_ZONES =
-      "https://sampleserver6.arcgisonline.com/arcgis/rest/services/WorldTimeZones/MapServer";
-  private static final String SERVICE_CENSUS =
-      "https://sampleserver6.arcgisonline.com/arcgis/rest/services/Census/MapServer";
-  private static final String SERVICE_RECREATION =
-      "https://sampleserver6.arcgisonline.com/arcgis/rest/services/Recreation/FeatureServer/0";
+  private FeatureLayer featureLayer;
 
   @Override
   public void start(Stage stage) {
 
     try {
-      // create stack pane and application scene
+      // create the stack pane and the application scene
       StackPane stackPane = new StackPane();
       Scene scene = new Scene(stackPane);
       scene.getStylesheets().add(getClass().getResource("/display_layer_view_state/style.css").toExternalForm());
 
-      // set title, size, and add scene to stage
+      // set a title, size, and add the scene to the stage
       stage.setTitle("Display Layer View State Sample");
       stage.setWidth(800);
       stage.setHeight(700);
       stage.setScene(scene);
       stage.show();
 
-      // create a control panel
-      VBox controlsVBox = new VBox(6);
-      controlsVBox.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(0,0,0,0.3)"), CornerRadii.EMPTY,
-          Insets.EMPTY)));
-      controlsVBox.setPadding(new Insets(10.0));
-      controlsVBox.setMaxSize(220, 90);
-      controlsVBox.getStyleClass().add("panel-region");
+      // create an ArcGISMap with the topographic basemap
+      ArcGISMap map = new ArcGISMap(Basemap.createTopographic());
+
+      // create a map view and set the ArcGISMap to it
+      mapView = new MapView();
+      mapView.setMap(map);
 
-      // create labels to display the view status of each layer
-      Label worldTimeZonesLabel = new Label("World Time Zones: ");
-      Label censusLabel = new Label("Census: ");
-      Label facilitiesLabel = new Label("Facilities: ");
+      // set the initial viewpoint for the map view
+      mapView.setViewpoint(new Viewpoint(new Point(-11e6, 45e5, SpatialReferences.getWebMercator()), 40000000));
 
-      worldTimeZonesLabel.getStyleClass().add("panel-label");
-      censusLabel.getStyleClass().add("panel-label");
-      facilitiesLabel.getStyleClass().add("panel-label");
+      // create a label to display the view status
+      Label layerViewStatusLabel = new Label("Current view status: ");
 
-      // add labels to the control panel
-      controlsVBox.getChildren().addAll(worldTimeZonesLabel, censusLabel, facilitiesLabel);
+      // create a checkbox UI that toggles the visibility of the feature layer
+      CheckBox visibilityCheckBox = new CheckBox();
+      visibilityCheckBox.setText("Layer active");
```

Comment:
now I understand this sample better, I think we should rename this to "Layer visibility"

Reply:
Making "layer visible" as discussed